ccontainermain.go

```diff
@@ -269,7 +271,7 @@ func shutdownCaché(inst string) (bool, error) {
 	}
 
 	// check that the shutdown was successful
-	if err := checkCmdOutcome("down", inst); err != nil {
+	if err := checkCmdOutcome("down", inst, false); err != nil {
 		log.Printf("Error: Caché was not shutdown successfully.\n")
```

Comment:
in the log.Printf() we need to review the comment. Not sure we can capture all possible cases. Maybe we should return the parsed info string...